images/win/windows2016.json

```diff
@@ -21,6 +21,7 @@
         "imagedata_file": "C:\\imagedata.json",
         "helper_script_folder": "C:\\Program Files\\WindowsPowerShell\\Modules\\",
         "psmodules_root_folder": "C:\\Modules",
+        "agent_tools_directory": "C:\\hostedtoolcache\\windows",
```

Comment:
You should also add `AGENT_TOOLSDIRECTORY` to environment variables like you have done in Win19 template

Reply:
Ah yep good catch, merged with main and added that.

https://github.com/actions/virtual-environments/pull/2457/files/8abd479475a4b4be8da50d1e4c7fc0b8853bf18b..1195d292eec2f950a1f8fdf1c8db3e183364532f